hoomd/md/angle.py

```diff
@@ -53,10 +53,10 @@ class Harmonic(Angle):
             The dictionary has the following keys:
 
             * ``k`` (`float`, **required**) - potential constant
-              (in units of energy/radians^2)
+              :math:`[\mathrm{energy} \cdot \mathrm{radians}^{-2}]`
```

Comment:
We document radians as an explicit unit here, but that is not done in any of hpmc (specifically for rotational move size arguments). Should we start documenting this everywhere?

Reply:
In 2D, the rotation move is in radians but I don't think it is in 3D. We could document it as arbitrary units for 3D.

Reply:
Thank you for this comment. I have one quick question related to this comment. 

Would I set unit as 

* ``k`` (`float`, **required**) - potential constant
:math:`[\mathrm{energy} \cdot \mathrm{radians}^{-2}]` for 2D and 
:math:`[\mathrm{energy} \cdot \mathrm{arbitrary angle unit}^{-2}]` for 3D

* ``t0`` (`float`, **required**) -  rest angle
:math:`[\mathrm{radians}]` for 2D and 
:math:`[\mathrm{arbitrary angle unit}]` for 3D

Or do you have any other suggestion or preference for unit style?

Reply:
No, don't change the units for the angle potentials. I was referring to the units of the `a` value in the HPMC integrator that @tommy-waltmann asked about.